CoreTableView/src/au/gov/asd/tac/constellation/views/tableview/components/ExportMenu.java

```diff
@@ -59,8 +58,8 @@
     private static final String EXPORT_CSV_FILE_CHOOSER_TITLE = "Export To CSV";
     private static final String EXPORT_XLSX_FILE_CHOOSER_TITLE = "Export To XLSX";
 
-    private static final String EXPORT_CSV_FILE_CHOOSER_DESCRIPTION = "CSV files (*" + FileExtensionConstants.COMMA_SEPARATED_VALUE + ")";
-    private static final String EXPORT_XLSX_FILE_CHOOSER_DESCRIPTION = "Excel files (*" + FileExtensionConstants.XLSX + ")";
+    public static final FileNameExtensionFilter CSV_FILE_FILTER = FileChooser.CSV_FILE_FILTER;
+    public static final FileNameExtensionFilter XLSX_FILE_FILTER = FileChooser.EXCEL_FILE_FILTER;
```

Comment:
unsure why these needed to be made public. But also given the value has now been set to, you could probably just use that value directly and remove the need to create a constant